config/services.yaml

```diff
@@ -0,0 +1,23 @@
+# This file is the entry point to configure your own services.
+# Files in the packages/ subdirectory configure your dependencies.
+
+# Put parameters here that don't need to change on each machine where the app is deployed
+# https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
+parameters:
+    env(MONGODB_URL): ''
```

Comment:
Why is it done this way vs. https://github.com/danskernesdigitalebibliotek/ddb-cover-service-importers/blob/develop/config/services.yaml#L8 ?

Reply:
This is the recommended approach:
https://symfony.com/doc/current/configuration/env_var_processors.html#built-in-environment-variable-processors